thumbfast.lua

```diff
@@ -41,20 +88,11 @@ mp.utils = require "mp.utils"
 mp.options = require "mp.options"
 mp.options.read_options(options, "thumbfast")
 
-if options.min_thumbnails < 1 then
-    options.min_thumbnails = 1
-end
-
-local os_name = ""
-
-math.randomseed(os.time())
-local unique = math.random(10000000)
-local init = false
+local script_location = mp.command_native({"expand-path", "~~/scripts/" .. mp.get_script_name() .. ".lua"})
```

Comment:
If I use https://mpv.io/manual/master/#options-script with the external path...

Reply:
It should be more robust now.